draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5032,6 +5082,11 @@ Content-Range: exampleunit 11.2-14.3/25
    through one or more proxies, which can then be secured using TLS
    (Transport Layer Security, <xref target="RFC8446"/>).
 </t>
+<t>
+   Because CONNECT changes the request/response nature of a HTTP connection,
```

Comment:
```suggestion
   Because CONNECT changes the request/response nature of an HTTP connection,
```

Reply:
Beyond that, I don't think this paragraph says anything useful. What does the reader need to know here?